1943 task add a button to deploy aws infra (#2022)
CGI-FR/IoT-Hub-Portal: README.md

* C2D Methods
* LoRA WAN device connectivity

> ## Quick Start

Reviewer: ```suggestion
## Quick Start

Ready to get started with IoT Hub Portal? Great! Follow these simple steps to quickly set up your environment:
```

> ### Azure

Reviewer: ```suggestion
### Step 1: Choose Your Cloud Provider
```

> [Quick Start](https://cgi-fr.github.io/IoT-Hub-Portal/vnext/azure/#quick-start) for Azure environment.

Reviewer: ```suggestion
IoT Hub Portal works seamlessly with both Azure and Amazon Web Services. Simply click on the link below that corresponds to your cloud provider of choice:

* For Azure users, follow the [Quick Start](https://cgi-fr.github.io/IoT-Hub-Portal/vnext/azure/#quick-start) guide.
* For Amazon Web Services users, follow the [Quick Start](https://cgi-fr.github.io/IoT-Hub-Portal/vnext/aws/#quick-start) guide.
```

> ### Amazon Web Services

Reviewer: ```suggestion
### Step 2: Configure Your Environment
```

> [Quick Start](https://cgi-fr.github.io/IoT-Hub-Portal/vnext/aws/#quick-start) for AWS environment.

Reviewer: ```suggestion
Once you've selected your cloud provider, you'll need to configure your environment by following the detailed instructions provided in the Quick Start guide. Don't worry, we've made it easy to follow along!

### Step 3: Start Using IoT Hub Portal

Congratulations, you're now ready to start using IoT Hub Portal! With our intuitive user interface and comprehensive set of features, you'll be able to easily manage your IoT devices and leverage the power of Azure IoT Hub or AWS IoT Core. Get started today!
```
